packages/@okta/vuepress-site/docs/release-notes/2025/index.md

```diff
@@ -6,6 +6,20 @@ title: Okta Classic Engine API release notes 2025
 
 ## January
 
+### Weekly release 2025.01.2
+
+| Change | Expected in Preview Orgs |
+|--------|--------------------------|
+| [Bugs fixed in 2025.01.2](#bugs-fixed-in-2025-01-2) | January 29, 2025 |
```

Comment:
you need to add authentication claims sharing here. It is for both Classic and OIE.